### PR TITLE
Bug/histogram table with plot

### PR DESCRIPTION
**Pull request by codymccoy**
added the histogram to the table with plot.

### CODE REVIEW

File: ViewModel/AggregatedStageDamage/CalculatedStageDamageVM.cs [hunk 1]
```diff
@@ -221,7 +221,7 @@ private void LoadComputedCurveRows(List<UncertainPairedData> computedCurves)
             for(int i =0;i<computedCurves.Count;i++)
             {
                 UncertainPairedData upd = computedCurves[i];
-                CurveComponentVM computeComponent = new CurveComponentVM(StringConstants.STAGE_DAMAGE, StringConstants.STAGE, StringConstants.DAMAGE);              
+                CurveComponentVM computeComponent = new ComputeComponentVM(StringConstants.STAGE_DAMAGE, StringConstants.STAGE, StringConstants.DAMAGE,DistributionOptions.HISTOGRAM_ONLY);
```
Comment (Brennan1994) [inline ViewModel/AggregatedStageDamage/CalculatedStageDamageVM.cs:224]: Careful. Richard renamed ComputeComponentVM to CurveComponentVM.

Comment (codymccoy) [reply ViewModel/AggregatedStageDamage/CalculatedStageDamageVM.cs:224]: Yeah, i saw that. I had some resolves that i worked through. I must have missed that one.

Comment (rnugent3) [reply ViewModel/AggregatedStageDamage/CalculatedStageDamageVM.cs:224]: I hope that was a reasonable name refactor.

Comment (Brennan1994) [reply ViewModel/AggregatedStageDamage/CalculatedStageDamageVM.cs:224]: It's a better name for sure. I never liked ComputeComponentVM. I just couldn't come up with something better. Something Something emotional invalidation, misnaming, and off by 1 errors, something something.

**Review by Brennan1994 — APPROVED** [review]

**Pull request merged**